sage/pallet/src/benchmarking.rs

```diff
@@ -0,0 +1,402 @@
+// Ajuna Node
+// Copyright (C) 2022 BlogaTech AG
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#![cfg(feature = "runtime-benchmarks")]
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use crate::{
+	mock::{
+		Balances, MockAccountId, MockAffiliatesFeeProvider, MockFilterHandler, MockSeasonManager,
+		MockTournamentFeeProvider, MockTransitionFeeProvider, MockTreasuryManager, RuntimeEvent,
+		RuntimeOrigin, SageBenchmarkHelper, System, Test, SEASON_ID_0,
+	},
+	Pallet as Sage, *,
+};
+use ajuna_primitives::{asset_manager::AssetInspector, fee_handler::GameFeeHandler};
+use example_transition::{
+	generic::ExampleTransitionGeneric,
+	types::{Asset, AssetId},
+};
+use frame_benchmarking::benchmarks_instance_pallet;
+use frame_support::parameter_types;
+use frame_system::RawOrigin;
+use sp_runtime::BuildStorage;
+
+parameter_types! {
+	pub const BenchPalletId: PalletId = PalletId(*b"sage/bch");
+}
+
+pub struct MockAssetMediatorBench;
+
+impl AssetManager for MockAssetMediatorBench {
```

Comment:
Hmm, I start to think that our mock runtime shouldn't use instances in the first place. It is not necessary, and we can just omit the redundant implementation here.

Reply:
Okay, I've removed the Instance from the mock runtime and adapted the tests and benchmarks for it